cclib/method/ddec.py

```diff
@@ -466,22 +520,22 @@ def condition_densities(self):
                     upperbigPhi = numpy.PINF
                     upperphi = numpy.PINF
 
-            iter = 0
-            while fitphi and iter < 50:
+            iteration = 0
+            while not phiconverged and iteration < 50:
                 # Flow diagram on Figure S1 in doi: 10.1039/c6ra04656h details the procedure.
-                iter = iter + 1
+                iteration = iteration + 1
                 midbigPhi = (lowerbigPhi + upperbigPhi) / 2.0
                 midphi = self._phiai(self._y_a[atomi], midbigPhi, atomi)[0]
                 # Exit conditions
-                if abs(lowerphi) < 1e-10:
+                if abs(lowerphi) < self.convergence_level:
                     bigphiAI[atomi] = lowerbigPhi
-                    fitphi = False
-                elif abs(upperphi) < 1e-10:
+                    phiconverged = True
+                elif abs(upperphi) < self.convergence_level:
                     bigphiAI[atomi] = upperbigPhi
-                    fitphi = False
-                elif abs(midphi) < 1e-10:
+                    phiconverged = True
+                elif abs(midphi) < self.convergence_level:
                     bigphiAI[atomi] = midbigPhi
-                    fitphi = False
+                    phiconverged = True
```

Comment:
If you break out of the loop here you could obviate some branching here.

Reply:
Yes. I've added break statements and reduced the indent here. Thanks.